server/src-lib/Hasura/Server/Telemetry.hs

```diff
@@ -142,12 +142,12 @@ computeMetrics sc =
       evtTriggers = Map.size $ Map.filter (not . Map.null)
                     $ Map.map tiEventTriggerInfoMap usrTbls
       rmSchemas   = Map.size $ scRemoteResolvers sc
-      funcs = Map.size $ Map.filter (not . fiSystemDefined) $ scFunctions sc
+      funcs = Map.size $ Map.filter (not . unSystemDefined . fiSystemDefined) $ scFunctions sc
```

Comment:
create an alias to `unSystemDefined` as `isSystemDefined` for better readability. And use everywhere else.